### DISCUSSION TITLE
@ functions are rough around the edges. Very hard to utilize.

### DISCUSSION THREAD

**Discussion by atljoseph** [cline/cline#429] [Feature Requests]
The @ functions are cool, but… So hard to use as it is right now. The file and folder list isn’t even filterable. How do you turn off what you selected and remain in the same task?? Not to mention, it takes time and attention to remember to type @ and select a command, and then manually select something else. It takes less time to see a checkbox in front of you and click it (keyboard shortcut!).  Not to mention, it is more discoverable that way. And these @ features are 100% distracting and useless if not paired with filtering based on user typing a martial match.

**Comment by tsdevau** [comment]
It does filter.  If you type `@` immediately followed by `partial folder or file name` it provides a list of filtered files and folders.

eg: `@myName` will show a list of files and folders containing _myName_(case insensitive).

Is that what you're referring to? Or am I not understanding correctly?
